base-notebook/Dockerfile

```diff
@@ -2,9 +2,7 @@
 # Distributed under the terms of the Modified BSD License.
 
 # Ubuntu 20.04 (focal)
-# https://hub.docker.com/_/ubuntu/?tab=tags&name=focal
-# OS/ARCH: linux/amd64
-ARG ROOT_CONTAINER=ubuntu:focal-20210416@sha256:86ac87f73641c920fb42cc9612d4fb57b5626b56ea2a19b894d0673fd5b4f2e9
+ARG ROOT_CONTAINER=ubuntu:focal
```

Comment:
```suggestion
ARG ROOT_CONTAINER=ubuntu:focal-20210416
```

Reply:
Ok makes sense, so were pinning to a date rather than a specific sha, I dig it :)